EndlessClient/Dialogs/GuildDialog.cs

```diff
@@ -151,8 +174,16 @@ private void SetState(GuildDialogState state)
                             SubText = _localizedStringFinder.GetString(EOResourceID.GUILD_CHANGE_YOUR_GUILD_DETAILS),
                             OffsetY = 45,
                         };
-                        modifyGuildItem.LeftClick += (_, _) => SetStateIfGuildMember(GuildDialogState.Modify);
-                        modifyGuildItem.RightClick += (_, _) => SetStateIfGuildMember(GuildDialogState.Modify);
+                        modifyGuildItem.LeftClick += (_, _) =>
+                        {
+                            _guildActions.GetGuildDescription(_characterProvider.MainCharacter.GuildTag);
```

Comment:
If the character isn't in a guild this should probably not be sent to the server. Maybe call `SetStateIfGuildMember(GuildDialogState.Modify)`, and if the guild dialog's state was changed then send the packet?

Reply:
Fixed